### PR TITLE
Adaptive Card update for VA

### PR DESCRIPTION
**Pull request by ryanisgrig**
### Conversational Experience
* New User Greeting
image

* Returning User Greeting
image

* Updated OnboardingDialog to only ask for name.
* Added MainDialogResponses to allow for dynamic text on greeting cards.
* Added dependency on using same image asset location/httprequest like Automotive/POI skills for accessing `wwwroot` hosted assets.
* Returning User Greeting card features select actions to trigger the selected skill. In the future, a VA can get these from skill manifests and allow for a random/preferred display of skills to try

## Checklist

- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added or updated the appropriate unit tests
- [X] I have tested any new/updated dialogs using Speech in the emulator to ensure the [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) property is set to enable a high quality speech-first experience and the appropriate [InputHints](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) are set correctly.

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/assistant/Dialogs/Main/Resources/MainDialogResponses.zh.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        "text": "我能帮上什么忙吗？",
+        "speak": "我能帮上什么忙吗？"
+      }
+    ],
+    "inputHint": "acceptingInput"
+  },
+  "ReturningUserGreetingTitle": {
+    "replies": [
+      {
+        "text": "欢迎回来 {Name}",
```
Comment (DingmaomaoBJTU) [inline solutions/Virtual-Assistant/src/csharp/assistant/Dialogs/Main/Resources/MainDialogResponses.zh.json:54]: > {Name} [](start = 21, length = 7)

Please remove the space before & after {}~

Comment (DingmaomaoBJTU) [inline solutions/Virtual-Assistant/src/csharp/assistant/Dialogs/Main/Resources/MainDialogResponses.zh.json:54]: >欢迎回来 {Name} [](start = 17, length = 11)

-> 欢迎回来{name}

File: solutions/Virtual-Assistant/src/csharp/assistant/Dialogs/Main/Resources/ReturningUser.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "actions": [
+        {
+            "type": "Action.OpenUrl",
+            "title": "Documentation",
+            "url": "https://aka.ms/customassistantdocs"
+        },
+      {
+        "type": "Action.OpenUrl",
+        "title": "Linked Accounts",
+        "url": "https://github.com/Microsoft/AI/blob/master/solutions/Virtual-Assistant/docs/virtualassistant-linkedaccounts.md"
```
Comment (darrenj) [inline solutions/Virtual-Assistant/src/csharp/assistant/Dialogs/Main/Resources/ReturningUser.json:42]: we should move these to aka.ms to protect against changes?

**Review by darrenj — CHANGES_REQUESTED** [review]

File: solutions/Virtual-Assistant/src/csharp/assistant/Dialogs/Main/Resources/MainDialogResponses.zh.json (hunk 1 above)
Comment (ryanisgrig) [reply solutions/Virtual-Assistant/src/csharp/assistant/Dialogs/Main/Resources/MainDialogResponses.zh.json:54]: done

**Review by darrenj — APPROVED** [review]

**Pull request merged**